core/src/com/cloud/agent/api/storage/StorageNfsVersionCommand.java

```diff
@@ -0,0 +1,44 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+package com.cloud.agent.api.storage;
+
+import com.cloud.agent.api.Command;
+
+public abstract class StorageNfsVersionCommand extends Command {
+
+    protected StorageNfsVersionCommand(){
```

Comment:
Actually, I kept it for the hierarchy. Classes extending <code>StorageNfsVersionCommand</code> will not compile if I remove it, in their constructors the error will be:
````
Implicit super constructor StorageNfsVersionCommand() is undefined. Must explicitly invoke another constructor
````
Do you agree about keeping it?